### PR TITLE
treetracker-denormalization

### PR DESCRIPTION
**Pull request by Kpoke**

### CODE REVIEW

**Comment by Kpoke** [comment]
@emmanue can you help set up the new database schema with terraform? And also review the Argo cd pull request

**Comment by dadiorchen** [comment]
@emmanue can you review this PR first? Because the terraform problem, I think we can hack the things out first for this issue.

**Comment by emmanue** [comment]
@dadiorchen  understood I will review this pr first.

File: applications/base/domain-migration-scripts.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    targetRevision: HEAD
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: domain-migration
+  syncPolicy:
+    syncOptions:
+      - CreateNamespace=true
+    automated:
+      selfHeal: true
+      prune: true
```
Comment (emmanue) [inline applications/base/domain-migration-scripts.yaml]: This yaml file needs to be removed.

File: applications/base/kustomization.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ resources:
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
   - images-api.yaml
+  - domain-migration-scripts.yaml
```
Comment (emmanue) [inline applications/base/kustomization.yaml]: remove this line too.

File: applications/overlays/development/domain-migration-scripts.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: domain-migration-app
+  namespace: argocd
+spec:
+  source:
+    path: deployment/overlays/development
```
Comment (emmanue) [inline applications/overlays/development/domain-migration-scripts.yaml]: This file needs to be removed too.

File: applications/overlays/development/kustomization.yaml [hunk 4]
```diff
@@ -11,6 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
+  - domain-migration-scripts.yaml
```
Comment (emmanue) [inline applications/overlays/development/kustomization.yaml]: This needs to be removed.

File: applications/overlays/production/domain-migration-scripts.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: domain-migration-app
+  namespace: argocd
+spec:
+  source:
+    path: deployment/overlays/production
```
Comment (emmanue) [inline applications/overlays/production/domain-migration-scripts.yaml]: This file needs to be removed.

File: applications/overlays/production/kustomization.yaml [hunk 6]
```diff
@@ -11,6 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
+  - domain-migration-scripts.yaml
```
Comment (emmanue) [inline applications/overlays/production/kustomization.yaml]: This line needs to be removed.

File: applications/overlays/test/domain-migration-scripts.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: domain-migration-app
+  namespace: argocd
+spec:
+  source:
+    path: deployment/overlays/test
```
Comment (emmanue) [inline applications/overlays/test/domain-migration-scripts.yaml]: This yaml file needs to be removed.

File: applications/overlays/test/kustomization.yaml [hunk 8]
```diff
@@ -11,6 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
+  - domain-migration-scripts.yaml
```
Comment (emmanue) [inline applications/overlays/test/kustomization.yaml]: This line needs to be removed.

**Review by emmanue — COMMENTED** [review]
The highlighted changes need to be made.

File: applications/base/kustomization.yaml [hunk 9]
```diff
@@ -8,7 +8,7 @@ resources:
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
   - images-api.yaml
-  - domain-migration-scripts.yaml
+  - treetracker-denormalization.yaml
```
Comment (emmanue) [inline applications/base/kustomization.yaml:11]: This line needs to be removed.

File: applications/base/treetracker-denormalization.yaml [hunk 10]
```diff
@@ -1,16 +1,16 @@
...
-    repoURL: https://github.com/Greenstand/domain-migration-scripts
+    repoURL: https://github.com/Greenstand/treetracker-denormalization
     targetRevision: HEAD
   destination:
     server: https://kubernetes.default.svc
-    namespace: domain-migration
+    namespace: treetracker-denormalization
   syncPolicy:
     syncOptions:
       - CreateNamespace=true
```
Comment (emmanue) [inline applications/base/treetracker-denormalization.yaml:16]: This yaml file needs to be removed.

File: applications/overlays/development/kustomization.yaml [hunk 11]
```diff
@@ -11,7 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
-  - domain-migration-scripts.yaml
+  - treetracker-denormalization.yaml
```
Comment (emmanue) [inline applications/overlays/development/kustomization.yaml:14]: This line needs to be removed.

File: applications/overlays/development/treetracker-denormalization.yaml [hunk 12]
```diff
@@ -1,8 +1,8 @@
...
 kind: Application
 metadata:
-  name: domain-migration-app
+  name: treetracker-denormalization
   namespace: argocd
 spec:
   source:
-    path: deployment/overlays/test
\ No newline at end of file
+    path: deployment/overlays/development
```
Comment (emmanue) [inline applications/overlays/development/treetracker-denormalization.yaml:8]: This file needs to be removed.

File: applications/overlays/production/kustomization.yaml [hunk 13]
```diff
@@ -11,7 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
-  - domain-migration-scripts.yaml
+  - treetracker-denormalization.yaml
```
Comment (emmanue) [inline applications/overlays/production/kustomization.yaml:14]: This line needs to be removed.

File: applications/overlays/production/treetracker-denormalization.yaml [hunk 14]
```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: domain-migration-app
+  name: treetracker-denormalization
   namespace: argocd
 spec:
   source:
```
Comment (emmanue) [inline applications/overlays/production/treetracker-denormalization.yaml:7]: This file needs to be removed.

File: applications/overlays/test/kustomization.yaml [hunk 15]
```diff
@@ -11,7 +11,7 @@ patchesStrategicMerge:
   - bulk-pack-processor.yaml
   - bulk-pack-transformer.yaml
   - bulk-pack-transformer-v2.yaml
-  - domain-migration-scripts.yaml
+  - treetracker-denormalization.yaml
```
Comment (emmanue) [inline applications/overlays/test/kustomization.yaml:14]: This line needs to be removed.

File: applications/overlays/test/treetracker-denormalization.yaml [hunk 16]
```diff
@@ -1,8 +1,8 @@
...
 kind: Application
 metadata:
-  name: domain-migration-app
+  name: treetracker-denormalization
   namespace: argocd
 spec:
   source:
-    path: deployment/overlays/development
\ No newline at end of file
+    path: deployment/overlays/test
```
Comment (emmanue) [inline applications/overlays/test/treetracker-denormalization.yaml:8]: This file needs to be removed.

**Review by emmanue — COMMENTED** [review]
Kindly check the review down,

File: applications/base/treetracker-denormalization.yaml [hunk 17]
```diff
@@ -1,16 +1,16 @@
...
 spec:
   project: default
   source:
-    repoURL: https://github.com/Greenstand/domain-migration-scripts
+    repoURL: https://github.com/Greenstand/treetracker-denormalization
     targetRevision: HEAD
   destination:
     server: https://kubernetes.default.svc
-    namespace: domain-migration
+    namespace: treetracker-denormalization
```
Comment (emmanue) [inline applications/base/treetracker-denormalization.yaml]: The namespace is: denormalization.

**Review by emmanue — COMMENTED** [review]
Please change the namespace, apart from that, everything else looks goods.

**Review by emmanue — APPROVED** [review]
Lgtm.

**Pull request merged**